storage/storage.go

```diff
@@ -253,6 +254,26 @@ func (storage DBStorage) ListOfClustersForOrg(orgID types.OrgID) ([]types.Cluste
 	return clusters, nil
 }
 
+// ListOfClustersForOrg reads list of all clusters fro given organization
+func (storage DBStorage) GetOrgIDByClusterID(cluster types.ClusterName) (types.OrgID, error) {
+	rows, err := storage.connection.Query("SELECT org_id FROM report WHERE cluster = $1 ORDER BY org_id LIMIT 1", cluster)
```

Comment:
Why do you have `order by` here?

Reply:
and also, you could probably use QueryRow

Reply:
> Why do you have `order by` here?

Basically it's just to be sure that we get first orgID, it can be easily removed, but I prefer to leave it as it is, just to be sure. 


> and also, you could probably use QueryRow

Yes, thank you

Reply:
it is unique according to DB schema

Reply:
will be nice to see execution plans for both alternatives

Reply:
is `order by` and `limit` there because of insufficient DB schema constraints?

Reply:
As I wrote before, I use `order by` just to be sure that we get first OrgId, even if by DB schema it will be unique.